.vscode/settings.json

```diff
@@ -83,5 +83,8 @@
     "subspec",
     "uifabricshared",
     "unemphasized"
-  ]
+  ],
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
esbenp?

Reply:
Yeah that's the name for the official Prettier extension.